### PR TITLE
Completed html-forms exercise. | Parker Siu

### PR DESCRIPTION
**Pull request by parkersiu**
Kapture 2022-10-11 at 13 31 13

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      Email
```
Comment (Zhiwutian) [inline html-forms/index.html:9]: you want the label for the input to wrap the text of the label.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <div>
+        <label for="email"><input id="email" type="email" name="email"></label>
+      </div>
+      Why is coding cool?
+      <div>
+        <select name="why">
```
Comment (Zhiwutian) [inline html-forms/index.html:15]: you need a label for this select input.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <label for="javascript">Javascript</label>
+      </div>
+      </p>
+      <p>Would you like to sign up for more information?
+      <div>
+        <label for="sign"><input id="sign" type="checkbox">Sign me up!</label>
+      </div>
+      Comments
+      <div>
+        <textarea name="comments"></textarea>
```
Comment (Zhiwutian) [inline html-forms/index.html:38]: dont forget the label for this textarea

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      </div>
+      <div>
+        <label for="submit"><input id="submit" type="submit" name="submit" value="Submit"></label>
```
Comment (Zhiwutian) [inline html-forms/index.html:41]: the only input you don't need a label for is the `submit` input

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a few updates are needed. Great work so far

**Comment by parkersiu** [comment]
> a few updates are needed. Great work so far

I have updated the labels per your comments.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**